sotodlib/mapmaking/ml_mapmaker.py

```diff
@@ -146,7 +155,7 @@ def transeval(self, id, obs, other, x, tod=None):
 
 class Signal:
     """This class represents a thing we want to solve for, e.g. the sky, ground, cut samples, etc."""
-    def __init__(self, name, ofmt, output, ext):
+    def __init__(self, name, ofmt, output, ext, glitch_flags: str = "flags.glitch_flags"):
```

Comment:
Does this need to be here?  Due to type checking issues?  Nothing is done with it...

Reply:
oh, sorry ... not sure how I didn't see that...  I was really looking for it...